akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -27,7 +28,10 @@ import scala.compat.java8.FutureConverters._
  * an “atomic” source, e.g. from a collection or a file. Materialization turns a Source into
  * a Reactive Streams `Publisher` (at least conceptually).
  */
-final class Source[+Out, +Mat](override val module: Module)
+final class Source[+Out, +Mat](
+  override val traversalBuilder: LinearTraversalBuilder,
```

Comment:
Hm, strange why it was public before. Maybe Gearpump needed it? We can make it hidden now as a custom phase will no longer need to know about the internal representation. Thanks for pointing out.